gbpservice/nfp/configurator/agents/loadbalancer_v1.py

```diff
@@ -737,5 +738,5 @@ def _start_collect_stats(sc):
 
 
 def init_agent_complete(cm, sc, conf):
-    _start_collect_stats(sc)
+    # _start_collect_stats(sc)
```

Comment:
Why is this commented? To avoid the context issue in update_pool_status? Can we fix and test this?